lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -137,6 +137,16 @@ CRI_F2F:
       targetState: F2F_HANDOFF_PAGE
     access-denied:
       targetState: PYI_ANOTHER_WAY
+    not-found:
+      targetState: ERROR
+    temporarily-unavailable:
```

Comment:
I'm curious how you found this - I think there is default behaviour here from the parent state which should be enough to pass the test(?)

If we're actively changing behaviour, I think we should probably separate that into a different PR - to keep this one as 'test only'.

Reply:
The 'end' case is particularly interesting - and we probably want to talk to some other people about whether an unexpected 'end' should just complete the journey, continue the journey as-is, or be an error (acknowledging that the user can probably just restart the journey and get a valid profile)

Reply:
In the case where we're using the parent CRI_STATE to set the target state of the end event, I think it should be ERROR, since we don't have a consistent action on end

Reply:
Could it expected where it relies on the default of going to IPV_SUCCESS_PAGE?

Reply:
Having talked to Patrick, since it shouldn't be reached we probably want to indicate this as an error so we can react to it. I'll update it to ERROR instead of the default IPV_SUCCESS_PAGE